op-devstack/dsl/ecotone_fees.go

```diff
@@ -58,24 +58,27 @@ func (ef *EcotoneFees) ValidateTransaction(from *EOA, to *EOA, amount *big.Int)
 	vaultsAfter := ef.getVaultBalances(client)
 	vaultIncreases := ef.calculateVaultIncreases(vaultsBefore, vaultsAfter)
 
-	l1Fee := big.NewInt(0)
```

Comment:
Overall comment for the `ValidateTransaction` method: There is a update for fee check. May I ask what kind of flakiness did you experience? Asking this because curious that this patch is towards reducing flakiness, or hardening test, or both.